### PR TITLE
add en-US/mozorg/internet-health/open-innovation.lang

### PR DESCRIPTION
**Pull request by peiying2**
per https://github.com/mozilla/bedrock/pull/4702

### CODE REVIEW

File: en-US/mozorg/internet-health/open-innovation.lang [hunk 1]
```diff
@@ -0,0 +1,249 @@
+
```
Comment (flodolo) [inline en-US/mozorg/internet-health/open-innovation.lang]: Why the empty line at the beginning of the file?

File: en-US/mozorg/internet-health/open-innovation.lang [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+## NOTE: Demo server is available at https://www-dev.allizom.org/internet-health/open-innovation/
+## URL: https://www-dev.allizom.org/%LOCALE%/internet-health/open-innovation/
+
+
+;Open Innovation: Vital to Internet Health
```
Comment (flodolo) [inline en-US/mozorg/internet-health/open-innovation.lang]: Explain that this is the HTML title of the page, and the next one is the description. You can check existing .lang files to see similar comments.

File: en-US/mozorg/internet-health/open-innovation.lang [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+Open source software – technology built with code that is open for view, use, and modification – is the engine that powers a huge amount of the Internet, from servers to operating systems to the bots that fetch your search results. It’s the infrastructure that makes the Web a truly public resource: transparent, trustworthy, and collaborative, so that anyone with an idea can contribute. But much like our IRL infrastructure, we have to commit the attention and resources to maintain it.
+
+
+;Github, the world’s largest open source community, includes 19 million developers, 100,000 teams, and 52 million repositories for established and developing technologies.
+Github, the world’s largest open source community, includes 19 million developers, 100,000 teams, and 52 million repositories for established and developing technologies.
```
Comment (flodolo) [inline en-US/mozorg/internet-health/open-innovation.lang]: Argh, this is wrong: GitHub. Will comment in the PR

File: en-US/mozorg/internet-health/open-innovation.lang [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+
+;Source: <a href="%(github)s">Github</a>
+Source: <a href="%(github)s">Github</a>
+
+
+# this is used as a decorative element in a callout box. Keep the translation short.
```
Comment (flodolo) [inline en-US/mozorg/internet-health/open-innovation.lang]: Don't keep original comments, explain what M is too.

File: en-US/mozorg/internet-health/open-innovation.lang [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+
+;There are now an estimated 1 billion Creative Commons-licensed works online that encourage reuse, including texts, photos, and music.
+There are now an estimated 1 billion Creative Commons-licensed works online that encourage reuse, including texts, photos, and music.
+
+
+;Source: <a href="%(cc)s">Creative Commons</a>, 2015
+Source: <a href="%(cc)s">Creative Commons</a>, 2015
+
+
+# this is used as a decorative element in a callout box. Keep the translation short.
```
Comment (flodolo) [inline en-US/mozorg/internet-health/open-innovation.lang]: Same here

File: en-US/mozorg/internet-health/open-innovation.lang (hunk 2 above)
Comment (peiying2) [inline en-US/mozorg/internet-health/open-innovation.lang]: I noticed I missed both notes in the digital-inclusion file, let me add to it too.

**Pull request merged**